### DISCUSSION TITLE
resolveProps in arrayfield

### DISCUSSION THREAD

**Discussion by tiagozaccaro** [data-driven-forms/react-forms#1283] [Need help]
how to use resolveProps in a input inside an arrayfield.

Exemple:

I have a arrayfield with 2 lines and in both lines I have (Price, Tax and Total)

How can I calculate total for each line based on Price and Tax?

Thanks

**Comment by Hyperkid123** [comment]
@tiagozaccaro if you want to listen (spy) on values and you want to have them up to date we recommend using the `FormSpy` component.

https://data-driven-forms.org/examples/value-listener#heading-formspyexample

I would probably see this as a new custom component, that would spy on the form state and calculate and display the value.

If you wish to integrate this sum into the field array, you will have to create a custom implementation for it. Proprietary props outside of the field array scope are nothing that is not possible to do the default field array components.

https://data-driven-forms.org/components/field-array#heading-fieldarray
